spectrum_utils/plot.py

```diff
@@ -286,3 +430,83 @@ def mirror(
     )
 
     return ax
+
+
+def full_mirror(
+    spec_top: MsmsSpectrum,
+    spec_mass_errors: Optional[MsmsSpectrum] = None,
+    spec_bottom: Optional[MsmsSpectrum] = None,
+    spectrum_kws: Optional[Mapping[str, Any]] = None,
+    mass_errors_kws: Optional[Mapping[str, Any]] = None,
+    height: Optional[float] = None,
+    ratio: Optional[float] = None,
+) -> plt.Figure:
+    """
+    Plot a spectrum, mass errors, and a mirror spectrum.
+
+    Parameters
+    ----------
+    spec_top : MsmsSpectrum
+        The spectrum to be plotted on the top.
+    spec_mass_errors : Optional[MsmsSpectrum], optional
+        The spectrum for which mass errors are to be plotted in the middle.
+    spec_bottom : Optional[MsmsSpectrum], optional
+        The spectrum to be plotted on the bottom.
+    spectrum_kws : Optional[Mapping[str, Any]], optional
+        Keyword arguments for `plot.spectrum` for the top and bottom spectra.
+    mass_errors_kws : Optional[Mapping[str, Any]], optional
+        Keyword arguments for `plot.mass_errors`.
+    height : Optional[float], optional
+        The height of the figure in inches.
+    ratio : Optional[float], optional
+        The width of the figure as a ratio of the height.
```

Comment:
**question:** Do you consider this easier to use than specifying the width in inches directly?

Reply:
Not sure. I replicated the Seaborn API for figure-level plots, as this might be more familiar to those who have already used Seaborn. What do you think?

(side note: I mistakenly used `ratio` instead of `aspect`, which has now been updated to be consistent with Seaborn)